packages/api/core/src/api/init-scripts/init-npm.ts

```diff
@@ -30,7 +31,22 @@ export default async (dir: string) => {
     const packageJSON = await readRawPackageJson(path.resolve(__dirname, '../../../tmpl'));
     // eslint-disable-next-line no-multi-assign
     packageJSON.productName = packageJSON.name = path.basename(dir).toLowerCase();
-    packageJSON.author = await username();
+
+    let author: string | undefined;
+
+    try {
+      const name = execSync('git config --get user.name', { encoding: 'utf8' }).trim();
+      const email = execSync('git config --get user.email', { encoding: 'utf8' }).trim();
+      author = `${name} <${email}>`;
```

Comment:
This should be an Object: https://docs.npmjs.com/files/package.json#people-fields-author-contributors